README.md

```diff
@@ -6,7 +6,8 @@
 2. Copy to tomcat/webapp
 3. Add the property file on linux box <code>sectionsToolPropsSecure.properties</code>,   
 then in JAVA_OPTS add the  
-<code>-DsectionsToolPropsPathSecure=file:/file-path/sectionsToolPropsSecure.properties</code>
+<code>-DsectionsToolPropsPathSecure=file:/file-path/sectionsToolPropsSecure.properties</code><br/>
+<code>-DsectionsToolFriendPropsPathSecure=file:/file-path/sectionsToolFriend.properties</code>
 4. Add the following 5 properties to sectionsToolPropsSecure.properties: 
 
     <code>canvas.admin.token=canvas token  
```

Comment:
it is good idea to keep 2 props file since both props are serving 2 different work flows